runtime/Cargo.toml

```diff
@@ -30,7 +30,7 @@ solana-measure = { path = "../measure", version = "0.18.0-pre0" }
 solana-metrics = { path = "../metrics", version = "0.18.0-pre0" }
 solana-bpf-loader-api = { path = "../programs/bpf_loader_api", version = "0.18.0-pre0" }
 solana-bpf-loader-program = { path = "../programs/bpf_loader_program", version = "0.18.0-pre0" }
-solana-noop-program = { path = "../programs/noop_program", version = "0.18.0-pre0" }
+solana-programs = { path = "../programs", version = "0.18.0-pre0" }
```

Comment:
This dep looks backwards.  Runtime can’t generically depend on programs

Reply:
I think you need two lists: (1) the minimal set that the runtime needs to function, “runtime-programs/“, and (2) everybody else, “core-programs”